prow/jobs/kyma-project/warden/warden-release.yaml

```diff
@@ -0,0 +1,2 @@
+# Code generated by rendertemplates. DO NOT EDIT.
+
```

Comment:
Why this file is empty?

I see proper target in template:
`- to: ../../prow/jobs/kyma-project/warden/warden-release.yaml`

Reply:
done